test/YieldDistributor.t.sol

```diff
@@ -451,6 +454,190 @@ contract YieldDistributorTest is Test {
     }
 }
 
+contract VotingStreakMultiplierTest is YieldDistributorTest {
+    // uint256 constant START = 32_323_232_323;
+    uint256 constant MULTIPLIER_INCREMENT = 0.02e20;
+    uint256 constant MAX_MULTIPLIER = 3;
+
+    // YieldDistributorTestWrapper public yieldDistributor;
```

Comment:
Personally not a fan of leaving in commented out stuff like this. Either we need it or we don't. Commented out stuff creates ambiguity.